Resources/public/Admin.js

```diff
@@ -199,22 +199,9 @@ var Admin = {
     },
 
     stopEvent: function(event) {
```

Comment:
Not sure of the utility of this function if it does not do this black magic for IE8